src/fusor/models.py

```diff
@@ -522,45 +484,41 @@ def enforce_element_quantities(cls, values):
             "Fusions must contain >= 2 structural elements, or >=1 structural element "
             "and a regulatory element"
         )
-        structural_elements = values.get("structural_elements", [])
-        if not structural_elements:
+        structure = values.get("structure", [])
```

Comment:
I don't see that `structural_elements` was renamed to `structure`. I thought it was renamed to `structuralElements` above

Reply:
thank you for catching this! that was my mistake - I changed it for Assayed/Categorical but not the overall AbstractFusion class. Going to update this when I push this next commit